consensus/src/pipeline/buffer_manager.rs

```diff
@@ -164,6 +165,10 @@ pub struct BufferManager {
     consensus_publisher: Option<Arc<ConsensusPublisher>>,
 
     pending_commit_proofs: BTreeMap<Round, LedgerInfoWithSignatures>,
+
+    // If the buffer manager receives a commit vote for a block that is not in buffer items, then
+    // the vote will be cached. We can cache upto 100 blocks, and upto 150 commit votes per block.
```

Comment:
Put the gap threshold (100) in the node config? The another number 150 seems unnecessary as said in the other comment.

Reply:
Done.